.changeset/shaggy-ravens-grab.md

```diff
@@ -0,0 +1,7 @@
+---
+'@api3/airnode-examples': patch
```

Comment:
Should be minor (you can edit the file directly).

Reply:
I changed all three, I hope this is correct?

Reply:
Yep.